scripts/build.py

```diff
@@ -163,3 +163,8 @@ def get_cache_slug():
     CSS_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', 'css', '')
     CSS_OUT_DIR = os.path.join(BUILD_DIR, 'css', '')
     copy_files_source_to_target(CSS_SRC_DIR, CSS_OUT_DIR)
+
+    # Copy core/templates/prod/head/ to build/[cache_slug]/templates/head/
+    TEMPLATES_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', '')
+    TEMPLATES_OUT_DIR = os.path.join(BUILD_DIR, 'templates', 'head', '')
```

Comment:
I think we could do this, i.e move everything to build/. We'll need to use regex + static_files in app.yaml to achieve this. This would require change to the build script (some of which is part of #2356) and we are also going to refactor the build process. So, I think we should do it separately.